src/poliastro/iod/izzo.py

```diff
@@ -30,9 +30,9 @@ def lambert(k, r0, r, tof, M=0, numiter=35, rtol=1e-8):
     rtol : float, optional
         Relative tolerance of the algorithm, default to 1e-8.
 
-    Yields
-    ------
-    v0, v : tuple
+    Returns
```

Comment:
This is incorrect, right? There's a `yield` below

Reply:
Thanks for pointing out!